src/modules/ce/components/project/ProjectCeReferralsPage.tsx

```diff
@@ -69,31 +69,30 @@ const ProjectCeReferralsPage: React.FC = () => {
     showReferralsTab,
   ]);
 
-  // const actions = useMemo(() => {
-  //   if (showOutgoingReferrals) {
-  //     return (
-  //       <ButtonLink
-  //         variant='contained'
-  //         to={generateSafePath(ProjectDashboardRoutes.SEND_REFERRAL, {
-  //           projectId: project.id,
-  //         })}
-  //       >
-  //         Send Referral
-  //       </ButtonLink>
-  //     );
-  //   }
-  // }, [project.id, showOutgoingReferrals]);
+  const actions = useMemo(() => {
+    if (showOutgoingReferrals) {
+      return (
+        <ButtonLink
+          variant='contained'
```

Comment:
Other Project pages with buttons in the top-right use a different style for buttons. Suggest updating style to match (outlined, possibly with icon if we have one?). See pages: Overview, Enrollments, Units, Inventory etc all follow this pattern.

Reply:
Sure! Maybe,
<img width="1130" height="304" alt="Screenshot 2025-07-23 at 5 37 21 PM" src="https://github.com/user-attachments/assets/481619b9-d1d5-4d00-b5c9-355c216be3e5" />
but I'll check in with design tomorrow to confirm